tensorflow/lite/tools/versioning/runtime_version.cc

```diff
@@ -125,6 +126,7 @@ std::string FindMinimumRuntimeVersionForOp(tflite::BuiltinOperator op_code,
            {{BuiltinOperator_FULLY_CONNECTED, 8}, "2.3.0"},
            {{BuiltinOperator_FULLY_CONNECTED, 9}, "2.3.0"},
            {{BuiltinOperator_FULLY_CONNECTED, 10}, "2.11.0"},
+           {{BuiltinOperator_FULLY_CONNECTED, 11}, "2.12.0"},
```

Comment:
@SaoirseARM
Could you update it to "2.13.0" like `BuiltinOperator_TRANSPOSE_CONV`?